cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -306,3 +309,110 @@ def test_alert_run_ansible_playbook(full_template_vm_modscope, alert_profile, re
             timeout=60)
     except TimedOutError:
         pytest.fail("Ansible playbook method hasn't been executed.")
+
+
+@pytest.fixture(scope='module')
+def setup_ansible_repository(appliance, wait_for_ansible):
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name="test_playbooks_automate",
```

Comment:
Hi Ganesh , what is playbook with same name already exists? 
Please change it to `name = fauxfactory.gen_alpha()`

Reply:
Added comment for repository name